### DISCUSSION TITLE
PIR senzors for light automations

### DISCUSSION THREAD

**Discussion by vednolacni** [ufodone/envisalink_new#59] [Q&A]
Hello,

i was wondering if its possible to turn lights on when there is motion detected?

**Comment by grunthos503** [comment]
Yes, your alarm motion sensors will work in Home Assistant just like any other motion sensor.  This is actually the main reason I bought an Envisalink.

If they show up as some other type of sensor, you can change the settings on the entity in HA to "Show As: Motion"

**Reply by vednolacni** [reply]
ok nice! i still have a problem my entitys (show as: motion) are always on (detectd). Do i need to program them in DSC alarm settings?
![pir history](https://github.com/ufodone/envisalink_new/assets/123391433/4961fb40-e860-4e21-bd72-3ff7014b4a04)
